.github/workflows/unittests.yaml

```diff
@@ -10,11 +10,11 @@ on:
 
 jobs:
   test:
-    runs-on: ubuntu-latest
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
         python-version: ["3.7", "3.8", "3.9", "3.10", "3.11.0-rc.1"]
```

Comment:
I think that the {`windows-latest`, `3.11.0-rc.1`} build [install-time failure for `lxml`](https://github.com/hhursev/recipe-scrapers/actions/runs/3221015714/jobs/5268442428#step:4:262) is due to the fact that no precompiled wheels are available for that {package, Python version, OS} combination on PyPi.

`macos-latest` did appear to manage to [install `lxml` successfully](https://github.com/hhursev/recipe-scrapers/actions/runs/3221015714/jobs/5268442100#step:4:172) under Py3.11-rc.1 (before the job was cancelled due to the Windows check failure)

Probably easiest to add an exclusion condition for the Windows + Py3.11-rc1 combination to resolve this.